bin/plugin/commands/changelog.js

```diff
@@ -541,7 +541,9 @@ function getEntry( issue ) {
 					title,
 					issue.number,
 					issue.html_url
-				);
+				) +
+				' - @' +
+				issue.user.login;
```

Comment:
Thanks for trying this, @getdave. I think listing the contributors participating in the release will improve the release page. However, I am not sure listing them next to each PR is the way we should go. I think it adds too granular info to the changelog, making it harder to read and focus on the PRs themselves. Moreover, copying the list of PRs alone elsewhere (Google Docs, make.wordpress.org/core) will need an extra step (e.g., a regex) to remove each author.

My two cents is adding the list of contributors to the bottom would be a nice middle ground. Later on, other participants could be added, apart from the person merging each PR.